internal/cmd/connect/command_plugin_install.go

```diff
@@ -146,14 +141,14 @@ func (c *pluginCommand) install(cmd *cobra.Command, args []string) error {
 	}
 
 	if dryRun {
-		output.Println("Skipping installation of connector onto worker as part of dry run mode.")
+		output.Println("Dry run: skipping installation of plugin.")
```

Comment:
```suggestion
		output.Println("Dry run: skipping plugin installation.")
```

Reply:
I think our other dry run text is formatted as "[DRY RUN] Message", let's double-check and stay consistent with that!